src/com/nkita/practice/collections/PersonStatisticHandler.java

```diff
@@ -219,4 +219,83 @@ public void checkOldestPerson(Gender gender) {
             System.out.println("The oldest Person(s) witch gender " + gender + "\n" + oldestPersons);
         }
     }
-}
\ No newline at end of file
+
+    public void checkSortedStatistic() {
+        List<Person> personList = new ArrayList<>(persons);
+        personList.sort(new Comparator<Person>() {
+            @Override
+            public int compare(Person age1, Person age2) {
+                if (age1.getAge() > age2.getAge()) {
+                    return -1;
+                } else if (age1.getAge() < age2.getAge()) {
+                    return 1;
+                } else {
+                    return 0;
+                }
+
+            }
+        });
+        Collections.sort(personList);
+
+        for (Person o : personList) {
+            System.out.println(o.getName() + " | " + o.getAge());
+        }
+    }
+
+    /**
+     * вывести в консоль всех пользователей кторые подходят под фильтры
+     *
+     * @param personFilters
+     */
+    public void checkFilteredStatistic(List<PersonFilter> personFilters) {
+        List<Person> personList = new ArrayList<>();
+        for (Person person : persons) {
+            if (checksTheFilterListAndTheSpecificPerson(personFilters, person)) {
+                personList.add(person);
+            }
+        }
+        printInfo(personList);
+    }
+
+    private boolean checksTheFilterListAndTheSpecificPerson(List<PersonFilter> personFilters, Person person) {
```

Comment:
If some method returns a boolean, it would be better to start the method name with words is/are/has/have/contains depending on what the method does.
for example this method could be renamed to 'isPersonValidForFilters'